imap_processing/tests/mag/test_mag_validation.py

```diff
@@ -256,11 +256,13 @@ def test_mag_l1b_validation(test_number, mocks):
         assert np.allclose(expected_time, mago_time, atol=1e-6, rtol=0)
 
 
-@pytest.mark.xfail(reason="All L1C edge cases are not yet complete")
-@pytest.mark.parametrize(("test_number"), ["013", "014", "015", "016"])
+@pytest.mark.parametrize(("test_number"), ["013", "014", "015", "016", "024"])
 @pytest.mark.parametrize(("sensor"), ["mago", "magi"])
 @pytest.mark.external_test_data
 def test_mag_l1c_validation(test_number, sensor):
+    if test_number != "024":
```

Comment:
Might as well put 013 and 014 in there as they pass!

```suggestion
    if test_number in ["013", "014", "024"]:
```